packages/neuron-ui/src/utils/validators/outputs.ts

```diff
@@ -10,7 +10,9 @@ export const validateOutputs = (
 ) => {
   for (let i = 0; i < items.length; i++) {
     try {
-      validateAddress(items[i].address || '', isMainnet)
+      if (items[i].address) {
```

Comment:
Only the last address is allowed to be empty because only the last fieldset can be set `send max`